tests/s3/copy_object/test_parallel_copy_object.py

```diff
@@ -478,64 +525,66 @@ def test_44804(self):
+            self.parallel_copy_object((self.src_bkt1, f"{obj_type}-obj1", self.des_bkt1,
+                                      f"{obj_type}-obj2"), (self.des_bkt1, f"{obj_type}-obj2",
+                                      self.src_bkt1, f"{obj_type}-obj1"))
+            resp = self.s3_obj.object_info(self.des_bkt1, f"{obj_type}-obj2")
             assert_utils.assert_equal(resp1[1]["ETag"], resp[1]["ETag"])
-            self.parallel_copy_object(("src-bucket1", f"{obj_type}-obj1", "dest-bucket1",
-                                      f"{obj_type}-obj2"),
-                                      ("dest-bucket1", f"{obj_type}-obj3", "src-bucket1",
-                                      f"{obj_type}-obj4"))
-            self.validate_copy_content("src-bucket1", f"{obj_type}-obj1",
-                                       "dest-bucket1", f"{obj_type}-obj2")
-            self.validate_copy_content("dest-bucket1", f"{obj_type}-obj3",
-                                       "src-bucket1", f"{obj_type}-obj4")
-            self.multi_parallel_copy_object(("dest-bucket1", f"{obj_type}-obj2", "src-bucket1",
-                                            f"{obj_type}-obj3"),
-                                            ("dest-bucket1", f"{obj_type}-obj2", "src-bucket1",
-                                            f"{obj_type}-obj4"),
-                                            ("dest-bucket1", f"{obj_type}-obj2", "dest-bucket2",
-                                            f"{obj_type}-obj4"),
-                                            numproc=3)
-            self.validate_copy_content("dest-bucket1", f"{obj_type}-obj2",
-                                       "src-bucket1", f"{obj_type}-obj3")
-            self.validate_copy_content("dest-bucket1", f"{obj_type}-obj2",
-                                       "src-bucket1", f"{obj_type}-obj4")
-            self.validate_copy_content("dest-bucket1", f"{obj_type}-obj2",
-                                       "dest-bucket2", f"{obj_type}-obj4")
+            self.parallel_copy_object((self.src_bkt1, f"{obj_type}-obj1", self.des_bkt1,
+                                      f"{obj_type}-obj2"), (self.des_bkt1, f"{obj_type}-obj3",
+                                      self.src_bkt1, f"{obj_type}-obj4"))
+            validate_copy_content(self.src_bkt1, f"{obj_type}-obj1", self.des_bkt1,
+                                  f"{obj_type}-obj2", s3_testobj=self.s3_obj,
+                                  down_path1=self.downld_path1, down_path2=self.downld_path2)
+            validate_copy_content(self.des_bkt1, f"{obj_type}-obj3", self.src_bkt1,
+                                  f"{obj_type}-obj4", s3_testobj=self.s3_obj,
+                                  down_path1=self.downld_path1, down_path2=self.downld_path2)
+            self.multi_parallel_copy_object((self.des_bkt1, f"{obj_type}-obj2", self.src_bkt1,
+                                            f"{obj_type}-obj3"), (self.des_bkt1,
```

Comment:
Line 566 : We can move the second tuple being passed to a new line here for readability like how its done in line no. 293 .
Also, we can indent the arguments passed inside the 1st tuple like how its done in line no. 292 .
Applicable to other calls to multi_parallel_copy_object as well.

Reply:
same as mentioned in other comment on similar lines